resolwe_bio/tools/parse_library_type.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+"""Return library type information from the Salmon output file."""
+import argparse
+import json
+
+
+def parse_arguments():
+    """Parse command line arguments."""
+    parser = argparse.ArgumentParser(description="Parse library type information.")
+    parser.add_argument('input_file', type=str, help="Salmon library type information file.")
+    return parser.parse_args()
+
+
+def main():
+    """Invoke when run directly as a program."""
+    args = parse_arguments()
+
+    with open(args.input_file) as infile:
+        data = json.load(infile)
+        if 'expected_format' in data and 'compatible_fragment_ratio' in data:
+            print(json.dumps(
+                {'strandedness': data['expected_format']},
+                separators=(',', ':'))
+            )
+            print(json.dumps(
+                {'fragment_ratio': round(float(data['compatible_fragment_ratio']), 2)},
+                separators=(',', ':'))
+            )
+        else:
+            print('{"proc.error":"Cannot parse library type information file."}')
```

Comment:
What exactly do you mean? Using printouts like this? `print('re-error my_errror_msg')` Do you have an example of where this is used like this? I tried to use it, but it does not evaluate in the process as I expected it would.

Reply:
https://github.com/genialis/resolwe-bio/blob/master/resolwe_bio/tools/genehcluster.py#L75

Reply:
Tnx!